### ISSUE TITLE
SEN12MS baseline

### ISSUE THREAD

**Issue by taeil** [Berkeley-Data/hpt#5]
[repo](https://github.com/Berkeley-Data/SEN12MS/tree/taeil)
[wandb](https://wandb.ai/cal-capstone/SEN12MS)

pretrained models are [downloaded](https://syncandshare.lrz.de/getlink/fiCDbqiiSFSNwot5exvUcW1y/trained_models) and placed on s3 (s3://sen12ms/pretrained_sup) for faster download along with other pre-trained models. 

1. use the pretrained models 
2. run various prediction and explore different metrics for baselines 

Once this is done, please convert this issue to the discussion for documentation purpose.

**Comment by taeil** [comment]
started new evaluation on the baseline pre-training models (supervised) 

[first one (single label) ](https://wandb.ai/cal-capstone/SEN12MS/runs/1wfixcci/overview?workspace=user-taeil )

**Comment by TsungChinHanKen** [comment]
single_label/single_DenseNet121_s1s2
single_label/single_DenseNet121_s2
single_label/single_ResNet50_RGB


the link is showing blank. can I check if this is the right link? thanks

**Comment by TsungChinHanKen** [comment]
- https://wandb.ai/cal-capstone/SEN12MS/table?workspace=user-taeil
- For both baseline and CLS training evaluation (from my perspective)
1. Must tracked metrics (loss, mircro f1, macro f1, top-1 accuracy, top-1 precision, top-1 recall, class-wise accuracy)
2. To be checked with Colorado that whether these needs to be reported ( top - {2...}acc, f{2..}, overall acc, average acc)

Thanks
